packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -919,7 +957,9 @@ export class ListItem
     const content = [
       this.renderContentStart(),
       this.renderCustomContent(),
+      this.renderIconStart(),
```

Comment:
Question: Do we always put the icons inside the custom content slots or vice versa? Just want to confirm for designs. cc @ashetland @SkyeSeitz

Reply:
Now that we have the icon props, we should be using them from Calcite icons. The content slots can be used for other custom content.

Reply:
Correct. my question was whether the custom content like an avatar surrounds the icons or goes inside of them.

Reply:
Sorry for the confusion. The content slots should be on the outside of the icons:
![CleanShot 2024-12-09 at 12 41 40@2x](https://github.com/user-attachments/assets/3b9ad51e-f089-4f00-af09-4a88f50d14a4)

Reply:
Sounds good. Code looks correct for that 👍